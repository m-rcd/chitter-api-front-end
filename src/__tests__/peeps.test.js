import React from 'react';
import renderer from 'react-test-renderer';
// import Enzyme, { shallow } from 'enzyme';
// import Adapter from 'enzyme-adapter-react-16';
import Peep from '../components/peeps';

// Enzyme.configure({ adapter: new Adapter() });

it('Peep Snapshot', () => {
  const tree = renderer.create(<Peep />).toJSON();
  expect(tree).toMatchSnapshot();
});

// describe('componentDidMount', () => {
//   beforeEach(() => {
//     fetch.resetMocks()
//   })
//
//   it('sets the state componentDidMount', async () => {
//     fetch.mockResponseOnce(JSON.stringify({body: 'hi'}))
//     fetch('https://chitter-backend-api.herokuapp.com/peeps').then(res => {
//       expect(res.body).toEqual('hi')
//     })
//
//     expect(fetch.mock.calls.length).toEqual(1)
//
//
//     const wrapper = shallow(<Peep/>)
//
//   })
// })
