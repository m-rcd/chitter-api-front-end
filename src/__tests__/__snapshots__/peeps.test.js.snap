// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Peep Snapshot 1`] = `
<div
  className="peeps"
/>
`;
